server/backends/migration_cache/migration_cache.go

```diff
@@ -232,3 +250,54 @@ func (mc *MigrationCache) Set(ctx context.Context, d *repb.Digest, data []byte)
 
 	return srcErr
 }
+
+func (mc *MigrationCache) copyDataInBackground(ctx context.Context) error {
+	for {
+		select {
+		case <-mc.quitChan:
+			return nil
+		case digestToCopy := <-mc.copyChan:
+			alreadyCopied, err := mc.dest.Contains(ctx, digestToCopy)
+			if err != nil {
```

Comment:
I would move the contents of this `case` block into a separate function which will allow you to freely use `return` to simplify the code.

Reply:
Done